src/modules/form/util/formUtil.tsx

```diff
@@ -1290,7 +1290,36 @@ const getMappedValue = (record: any, mapping: FieldMapping) => {
     );
     return customDataElementValueForKey(mapping.customFieldKey, cdes);
   } else if (mapping.fieldName) {
-    return get(record, compact([relatedRecordAttribute, mapping.fieldName]));
+    const keys = compact([relatedRecordAttribute, mapping.fieldName]); // for example: ['disabilityGroup', 'viralLoadSource']
+
+    // FIXME - do these special cases make sense or should they be fixed instead of carved out here?
+    // Special cases where we DON'T want to raise an error if the field can't be resolved on the record:
+    // 1. The only key in `record` is `enrollment`. This happens in AssessmentForm
+    // around line 212 when we are creating a new assessment and populating it from the enrollment, but other
+    // fields that are normally resolved on an Assessment, like incomeBenefit, disabilityGroup, etc. are not resolved.
+    const recordKeys = Object.keys(record);
+    if (recordKeys.length === 1 && recordKeys[0] === 'enrollment') {
```

Comment:
Notes from dev standup:
- add a flag `dontRaise` to the method
- see custom fields that we process onto something else - like MCI
- avoid huge investment if we are moving into backend
- replace error with reporting to Sentry